CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Ensure that static frameworks are not embedded
```

Comment:
nit add two empty spaces at the end of this sentence for markdown formatting.

Reply:
Cheers, done.